### ISSUE TITLE
No support for BigInt

### ISSUE THREAD

**Issue by mzedeler** [glenjamin/transit-immutable-js#53]
```
const { toJSON } = require('transit-immutable-js')
const { Map } = require('immutable')
toJSON(Map().set(1n, 'test')
```
This results in `Uncaught TypeError: Cannot use 'in' operator to search for 'toMap' in 1`

The same goes for `toJSON(Map().set('test', 1n))`.

I may be able to add BigInt support if you're open for PRs.'

**Comment by glenjamin** [comment]
Yep, I’ll take a PR - as long as it gracefully skips registering when BigInt is not available.

Alternatively you should be able to add a custom decoder in your application
